src/armor_layers.cpp

```diff
@@ -300,8 +300,6 @@ std::vector<std::string> clothing_properties(
                                      width ) );
     props.push_back( name_and_value( space + _( "Warmth:" ),
                                      string_format( "%3d", worn_item.get_warmth() ), width ) );
-    props.push_back( name_and_value( space + string_format( _( "Storage (%s):" ), volume_units_abbr() ),
```

Comment:
Should this generate strings for pocket data?

Reply:
you mean item_contents::info()? or do you mean something else?

Reply:
I mean in `clothing_properties`. This removes the storage info from the armor layer menu and I'm thinking it should be replaced with pockets info.

Reply:
i've decided that this could be a follow-up pr to specfically improve UI and UI